docs/tutorials/working_with_large_catalogs.ipynb

```diff
@@ -2,22 +2,395 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    other_degree_surveys_path = "https://epyc.astro.washington.edu/~lincc-frameworks/other_degree_surveys"
</pre><p>This can just be <code>surveys_path</code>,the full name I think is confusing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/236/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>